client/src/components/Sidebar/Sidebar.js

```diff
@@ -29,11 +29,27 @@ const Sidebar = (props) => {
       <CurrentUser />
       <Typography className={classes.title}>Chats</Typography>
       <Search handleChange={handleChange} />
-      {conversations
-        .filter((conversation) => conversation.otherUser.username.includes(searchTerm))
-        .map((conversation) => {
-          return <Chat conversation={conversation} key={conversation.otherUser.username} />;
-        })}
+      {searchTerm !== ""
+        ? conversations
+            .filter((conversation) =>
+              conversation.otherUser.username.includes(searchTerm)
+            )
+            .map((conversation) => {
+              return (
+                <Chat
+                  conversation={conversation}
+                  key={conversation.otherUser.username}
+                />
+              );
+            })
+        : conversations.map((conversation) => {
+            return (
+              <Chat
+                conversation={conversation}
+                key={conversation.otherUser.username}
+              />
+            );
+          })}
```

Comment:
Same comment as above about only including relevant changes

Reply:
is it ok for me to leave it there for future PRs, or do you want me to revert it back to its original file ?